src/main/kotlin/org/jetbrains/research/testspark/tools/kex/generation/KexProcessManager.kt

```diff
@@ -0,0 +1,350 @@
+                javaExecPath,
+                projectContext.cutModule!!,
+                target,
+                resultName,
+                kexSettingsState,
+                kexExecPath,
+                kexHome,
+            ).buildCommand()
+
+            val cmdString = cmd.fold(String()) { acc, e -> acc.plus(e).plus(" ") }
+            log.info("Starting Kex with arguments: $cmdString")
+
+            // run kex as subprocess
+            runKex(cmd, errorMonitor, indicator)
+
+            log.info("Save generated test suite and test cases into the project workspace")
+
+            preprocessGeneratedTestFiles(resultName, classFQN, generatedTestsData, projectContext, errorMonitor)
+        } catch (e: Exception) {
+            kexErrorManager.errorProcess(
+                KexMessagesBundle.get("KexErrorCommon").format(e.message),
+                project,
+                errorMonitor,
+            )
+            e.printStackTrace()
+        }
+
+        return UIContext(
+            projectContext,
+            generatedTestsData,
+            StandardRequestManagerFactory(project).getRequestManager(project),
+            errorMonitor,
+        )
+    }
+
+    private fun preprocessGeneratedTestFiles(
+        resultName: String,
+        classFQN: String,
+        generatedTestsData: TestGenerationData,
+        projectContext: ProjectContext,
+        errorMonitor: ErrorMonitor,
+    ) {
+        val report = IJReport()
+        val imports = mutableSetOf<String>()
+        val packageStr = classFQN.substringBeforeLast('.')
+
+        val generatedTestsDir = File(
+            "$resultName/tests/${classFQN.substringBeforeLast('.').replace('.', '/')}")
+        if (generatedTestsDir.exists() && generatedTestsDir.isDirectory) { // collect all generated tests into a report
+            for ((index, file) in generatedTestsDir.listFiles()!!.withIndex()) {
+                val testCode = file.readText()
+
+                if (file.name.contains("Equality") || file.name.contains("Reflection")) {
+                    // collecting all methods
+                    generatedTestsData.otherInfo += "${getHelperClassBody(testCode)}\n"
+                } else {
+                    // merge @before and @test annotated methods into a single method
+                    report.testCaseList[index] =
+                        TestCase(
+                            index,
+                            file.name.substringBefore('.'),
+                            extractTestMethod(file, index).toString(),
+                            setOf(),
+                        )
+                }
+                // extracting just imports out of the test code
+                // remove imports from helper classes
+                imports.addAll(
+                    getImportsCodeFromTestSuiteCode(testCode, projectContext.classFQN!!)
+                        .filterNot {
+                            it.contains("import static org.example.EqualityUtils.*;") ||
```

Comment:
What is the purpose of this filer, and why do we use `org.example.` package here?